src/ComputerPlayer.java

```diff
@@ -4,8 +4,12 @@ public class ComputerPlayer extends Player{
 	
 	public ComputerPlayer(){
 		super("R2D2");
+		// #5, have to actually set the name equal to what you 
+		// want it to be 
+		myName = "R2D2";
```

Comment:
This fixes the error for the ComputerPlayer but not for players in general.  Look at superclass.  In particular, the name gets PASSED to the super constructor, so that's where the name setting should happen.